src/main/java/seedu/address/model/person/Birthday.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+/**
+ * Represents a Person's birthday in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidBirthday(String)}
+ */
+public class Birthday {
+
+    public static final String MESSAGE_BIRTHDAY_CONSTRAINTS =
+            "Person's birthday should be in the format of DDMMYYYY, and it should not be blank";
+
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String BIRTHDAY_VALIDATION_REGEX = "\\d{2}\\d{2}\\d{4}";
```

Comment:
I think \\d{8} equals to \\d{2}\\d{2}\\d{4}?

And... do we need to check DDMMYYYY, DD is in the range 01 - 31, MM is in the range 01-12, YYYY is in the range 1900 - 201x? today
I think maybe VALIDATION_REGEX=([0-3])([0-9])([0,1])([0-9])([1-2])(//d{3}),
what do you think? :)